types/api/network.d.ts

```diff
@@ -88,7 +95,7 @@ declare namespace my {
   /**
    * 小程序网络请求
    */
-  function request(options: IHttpRequestOptions): void;
+  function request(options: IHttpRequestOptions): IRequestPromise;
```

Comment:
In the [document](https://opendocs.alipay.com/mini/api/owycmh#mPip1), the return type of `request` called `RequestTask`.

Reply:
Changes on this part updated.